c-scape/src/c.rs

```diff
@@ -894,14 +893,7 @@ pub unsafe extern "C" fn memrchr(s: *mut c_void, c: c_int, len: usize) -> *mut c
 pub unsafe extern "C" fn memcmp(a: *const c_void, b: *const c_void, len: usize) -> c_int {
```

Comment:
Compiler-builtins already directly exports these functions unmangled.

Reply:
Ah, right, so we can't also export them unmangled. I don't think we can just re-export the `compiler_builtins` functions because the signatures are different, they use pointers to `u8` instead of `c_void`. What would you suggest doing instead?

Reply:
`compiler_builtins` has a `mangled-names` feature, so use that I guess.

Reply:
> I don't think we can just re-export the compiler_builtins functions because the signatures are different, they use pointers to u8 instead of c_void.

I don't think it is necessary to re-export. If you need to call it you could decalre their signature in an `extern "C" {}` block.

Reply:
Would that be preferable to having `compiler_builtins` mangle its names and exporting our own wrappers with the right signatures?

Reply:
I think so.